README.adoc

```diff
@@ -44,9 +36,10 @@ image::{model}[]
 image::{example}[width=600]
 
 .Example Query:
-[source,cypher,subs=attributes]

```

Comment:
you might want to keep `subs=attributes` as well, it might be used by some other system?